catalogue/webapp/pages/works.js

```diff
@@ -335,9 +329,8 @@ const Works = ({ works, searchParams }: Props) => {
                         />
                       </div>
                       <TogglesContext.Consumer>
-                        {({ relevanceRating, relevanceRatingOptIn }) =>
-                          relevanceRating &&
-                          relevanceRatingOptIn && (
+                        {({ relevanceRating }) =>
```

Comment:
Assuming we need to remove both?

Reply:
No, (it's a bit confusing) - relevanceRatingOptIn was used to show/hide the entire opt in and rating functionality. That's what we've now removed.

relevanceRating is used to show/hide the rating stars depending on if a user has opted in or not, so we still need that